suzieq/config/schema/topology.avsc

```diff
@@ -57,7 +57,7 @@
         },
         {
             "name": "asn",
-            "type": "string",
+            "type": "long",
```

Comment:
There is also the peerAsn field which is a string instead of a long. Should we update that as well?

Reply:
Yes! I missed that. Thanks